src/common/url/parse-url.js

```diff
@@ -32,6 +32,7 @@ export function parseUrl (url) {
     try {
       urlEl = new URL(url, location.href)
     } catch (err) {
+      // IE doesn't support URL(), so for IE non-web contexts (e.g. workers), `parseUrl` will return the default `{}`.
```

Comment:
This is not entirely accurate. The if block checks for a browser scope and uses an anchor element which should be supported in all browsers we support + IE11. This else block is only going to get called in a now browser scope like a worker and we do not support workers in IE11.

Reply:
The comment is a bit confusing and not super relevant, so removing it. Thanks

Reply:
The catch block could still be considered relevant just in case what gets passed to `URL` constructor is not a valid URL. That will cause an exception to be thrown.